hack/tests/subcommand.sh

```diff
@@ -30,14 +30,28 @@ kubectl create namespace test-memcached
 operator-sdk test local ./test/e2e --up-local --namespace=test-memcached --kubeconfig $KUBECONFIG
 kubectl delete namespace test-memcached
 
+# test operator in up local mode with namespace set to ""
+# --namespace="" is allowed in local mode only when --no-setup is set
+kubectl create namespace test-memcached
```

Comment:
Why you are creating the namespace if it is not used?

Reply:
true, it can be avoided.